<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmFromToFrame</class>
 <widget class="QWidget" name="frmFromToFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>71</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblFrom">
       <property name="text">
        <string>From Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtFrom"/>
     </item>
     <item>
      <widget class="QLabel" name="lblTo">
       <property name="text">
        <string>To Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtTo"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>258</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
